testng-eclipse-plugin/src/main/org/testng/eclipse/util/CustomSuite.java

```diff
@@ -250,34 +253,28 @@ public File save(File directory) {
     return suiteFile;
   }
 
-  protected void saveSuiteContent(final File file, final XMLStringBuffer content) {
-    FileWriter fw= null;
-    BufferedWriter bw= null;
+  protected void saveSuiteContent(File file, XMLStringBuffer content) {
+    FileOutputStream fileOutputStream = null;
+    BufferedOutputStream bufferedOutputStream = null;
+    OutputStreamWriter osw = null;
     try {
-      fw= new FileWriter(file);
-      bw= new BufferedWriter(fw);
-      bw.write(content.getStringBuffer().toString());
-      bw.flush();
-    }
-    catch(IOException ioe) {
-    }
-    finally {
-      if(null != bw) {
+        fileOutputStream = new FileOutputStream(file);
+        bufferedOutputStream = new BufferedOutputStream(fileOutputStream);
+        // use utf8 to solve special character problem
+        osw = new OutputStreamWriter(bufferedOutputStream, Charset.forName("UTF-8"));
+        osw.write(content.getStringBuffer().toString());
+    } catch (IOException ioException) {
+        // ignore ioException
+    } finally {
         try {
-          bw.close();
+            osw.close();
+            bufferedOutputStream.close();
+            fileOutputStream.close();
+        } catch (Exception e) {
+          // ignore
         }
-        catch(IOException ioe) {
-        }

```

Comment:
haha. should I use "TestNGPlugin.log(e.getStatus());" or "e.printStackTrace();" ?
"e.printStackTrace();" may be enough?